src/core.jl

```diff
@@ -20,6 +20,27 @@ struct ImageEdge
     weight::Float64
 end
 
+# TODO: add methods via dispatch for accessing fields of `FuzzyCMeansResult`
+# Accessor functions
+"Returns an array containing the assigned labels"
```

Comment:
Better to also show the call:
```julia
"""
    img_labeled = labels_map(seg)

Return an array containing the label assigned to each pixel.
"""
```
otherwise there is no indication of how many input/output arguments this function has.